tests/api/test_isolation_group_member.py

```diff
@@ -0,0 +1,54 @@
+from pprint import pprint
+
+
+class TestSaiIsolationGroupMember:
+    # object with parent SAI_OBJECT_TYPE_ISOLATION_GROUP SAI_OBJECT_TYPE_PORT SAI_OBJECT_TYPE_BRIDGE_PORT
+
+    def test_isolation_group_member_create(self, npu):
+        commands = [
+            {
+                'name': 'isolation_group_1',
+                'op': 'create',
+                'type': 'SAI_OBJECT_TYPE_ISOLATION_GROUP',
+                'attributes': [
+                    'SAI_ISOLATION_GROUP_ATTR_TYPE',
+                    'SAI_ISOLATION_GROUP_TYPE_PORT',
+                ],
+            },
+            {
+                'name': 'port_1',
```

Comment:
Is expected to be create on switch init. Please use objects discovery logic instead.

Reply:
added the fix